Lib/typing.py

```diff
@@ -2239,6 +2239,12 @@ def Annotated(self, *params):
 
         assert Annotated[Annotated[T, Ann1, Ann2], Ann3] == Annotated[T, Ann1, Ann2, Ann3]
 
+      However, this does not apply to Annotated types referenced through
```

Comment:
Please, explain why: since it will trigger a `TypeAliasType` evaluation.

Reply:
Done!